tests/benchdnn/doc/knobs_attr.md

```diff
@@ -8,8 +8,9 @@
     --attr-rounding-mode=ARG:MODE[+...]
     --attr-deterministic=BOOL
     --attr-dropout=PROBABILITY[:SEED[:TAG]]
-    --attr-scales=ARG:POLICY[:SCALE[:DATA_TYPE[:GROUPS]]][+...]
-    --attr-zero-points=ARG:POLICY[:ZEROPOINT[:DATA_TYPE[:GROUPS]]][+...]
+    --attr-scales=ARG:POLICY[:SCALE][:DATA_TYPE[:GROUPS]][+...]
+    --attr-zero-points=ARG:POLICY[:ZEROPOINT][:DATA_TYPE[:GROUPS]][+...]
+    --attr-precomputed-reductions=ARG:POLICY:DATA_TYPE:GROUPS[+...]
```

Comment:
should this be accompanied by extending inputs..?

Reply:
For a generally applicable feature - yes. In this case I refrain from extending them as its usability is very limited. Test cases will come from OV later once they integrate the functionality and update their verbose logs.

Reply:
should we at least add a toy case to ensure benchdnn/verbose/etc. work as expected?

Reply:
Would it make sense to add this option to the self check?